[Document-7870][docs] Modify the document system screenshot

amaoisnb opened the pull request:
- Upgrade 3.0 and dev branch screenshot to laster
- Upgrade Workflow Definition page

amaoisnb committed:
[Document-7870][docs]Modify the system screenshot
Upgrade 3.0 and dev branch screenshot to laster
Upgrade Workflow Definition page

zhongjiajie commented:
thanks, and will task a look tonight

zhongjiajie commented:
Hi @amaoisnb we only need to use the screenshot in English in our doc, so you should change the file https://raw.githubusercontent.com/apache/dolphinscheduler/12fdd9bbd9d15dfe2a993731a2b5054ea5eb3956/docs/docs/zh/guide/start/quick-start.md and make them using the same picture in English version document.

and remove the exists img in path `img/dag1.png` and `img/task-log.png`

amaoisnb committed:
[Document-7870][docs]Modify the system screenshot
Upgrade 3.0 and dev branch screenshot to laster
Upgrade Workflow Definition page

zhongjiajie commented:
@amaoisnb change LGTM, but could you resolve the conflicts

amaoisnb commented:
> @amaoisnb change LGTM, but could you resolve the conflicts
Okay, I'll try

zhongjiajie committed:
Merge branch 'dev' into Document-7870

zhongjiajie commented:
@amaoisnb I solve conflict by myself, hope you do not mind

EricGao888 commented on docs/docs/zh/guide/start/quick-start.md:
```diff
@@ -50,11 +50,11 @@
 
 <p align="center">
-   <img src="../../../../img/dag1.png" width="60%" />
+   <img src="../../../../img/process_definition_en.png" width="60%" />
```
As we have discussed here before: https://github.com/apache/dolphinscheduler/issues/10257#issuecomment-1175126227, what about renaming `process_definition_en` to `workflow_definition_en`?

zhongjiajie replied:
do you mean rename `process_definition_en` to `process_definition` ?

zhongjiajie replied:
> do you mean rename `process_definition_en` to `process_definition` ?

oh, my bad, you mean `process_definition` vs `workflow` ? we should do it, but I think it is better to do it in separate PR, and change other related content, WDYT

EricGao888 replied:
Sure, sounds good to me.
